src/Tools/GenerateGlobalAnalyzerConfigs/Program.cs

```diff
@@ -379,6 +379,18 @@ static string GetPackageSpecificContents(string packageName)
 	<AdditionalFiles Include=""AnalyzerReleases.Unshipped.md"" />
   </ItemGroup>";
                 }
+                else if (packageName == "Microsoft.CodeAnalysis.PublicApiAnalyzers")
+                {
+                    return @"
+
+  <!-- Workaround for https://github.com/dotnet/roslyn/issues/4655 -->
+  <ItemGroup Condition=""Exists('$(MSBuildProjectDirectory)\PublicAPI.Shipped.txt')"" >
+	<AdditionalFiles Include=""PublicAPI.Shipped.txt"" />
+  </ItemGroup>
+  <ItemGroup Condition=""Exists('$(MSBuildProjectDirectory)\PublicAPI.Unshipped.txt')"" >
+	<AdditionalFiles Include=""PublicAPI.Unshipped.txt"" />
+  </ItemGroup>";
```

Comment:
This is going to cause a problem for projects that already include these files.

Reply:
I will change the condition to also check if it is not already an additional file.

Reply:
I verified that the additional files get de-duped somewhere in the pipeline. Analyzer only sees unique additional files. For example, I added the following to a project and verified under the debugger that unique additional files are present in analyzer options:
```xml
	<ItemGroup>
		<AdditionalFiles Include="PublicAPI.Shipped.txt" />
		<AdditionalFiles Include="PublicAPI.Shipped.txt" />
	</ItemGroup>
	<ItemGroup>
		<AdditionalFiles Include="PublicAPI.Unshipped.txt" />
		<AdditionalFiles Include="PublicAPI.Unshipped.txt" />
	</ItemGroup>
```

Reply:
Adding items to an item group excludes duplicates by default though I'm not positive that is true both inside and outside a target. @rainersigwald❔

See also https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-items?view=vs-2019#BKMK_KeepDuplicates

Reply:
Could this cause problems in projects with code like we have at https://github.com/dotnet/aspnetcore/blob/0437117cfb5bc91a34878fb962076bb05f4b8273/eng/targets/CSharp.Common.targets#L27-L30 i.e. when exactly is this program run and what's the default `*.targets` file it'll create❔

Reply:
This is added to the targets file that is included with the NuGet package:
![image](https://user-images.githubusercontent.com/10605811/88710091-f7590900-d0ca-11ea-910d-7b36f6fe0656.png)

![image](https://user-images.githubusercontent.com/10605811/88710207-266f7a80-d0cb-11ea-935d-ea1f51c0d833.png)

Reply:
Adding items creates duplicates by default both inside and outside targets; only inside targets with that attribute does it filter.

Reply:
Oops. Thanks @rainersigwald❕